pySDC/projects/PinTSimE/estimation_check.py

```diff
@@ -239,15 +242,19 @@ def plotStateFunctionAroundEvent(u_num, prob_cls_name, M_fix):  # pragma: no cov
 
                     if use_SE:
                         t_switches = [u_num[dt][M_fix][use_SE][use_A]['t_switches'] for dt in dt_list]
-                        t_switch = [t_event[i] for t_event in t_switches]
+                        for t_switch_item in t_switches:
+                            for m in range(1, len(t_switch_item)):
+                                if np.isclose(t_switch_item[m - 1], t_switch_item[m], atol=1e-10):
+                                    t_switch_item.pop(m - 1)
 
+                        t_switch = [t_event[i] for t_event in t_switches]
                         ax[0, ind].plot(
                             dt_list,
                             [
                                 h_item[m]
                                 for (t_item, h_item, t_switch_item) in zip(t, h, t_switch)
                                 for m in range(len(t_item))
-                                if abs(t_item[m] - t_switch_item) <= 1e-14
+                                if abs(t_item[m] - t_switch_item) <= 2.7961188919789493e-11
```

Comment:
How did you arrive at this number? :D You could use something like the 10 times the `eps` attribute of float (see [here](https://numpy.org/doc/stable/reference/generated/numpy.finfo.html)) if that makes sense?

Reply:
I tried that out.. but for the current implementation of the SE I don't know if such an statement makes sense anymore. Since you use the $\alpha\leq 1$ (and probably $\alpha<1$) the event time ``t_switch_item`` here lies more far away from the logged time(?)